src/bundle/Controller/UserSettingsController.php

```diff
@@ -104,14 +104,23 @@ public function updateAction(Request $request, UpdateView $view)
                     'ibexa_user_settings'
                 );
 
+                $route = $request->query->get('route') ?? null;
+                $routeParameters = $request->query->get('routeParameters') ?? [];
```

Comment:
`InputBag::get` accepts default value as a second argument. It's `null` by default.
```suggestion
                $route = $request->query->get('route');
                $routeParameters = $request->query->get('routeParameters', []);
```

Reply:
Additionally, `routeParameters` should be acquired by `QueryBag::all()` method.

Reply:
Fixed in bd24d6e2.